pandas/plotting/_core.py

```diff
@@ -1935,52 +1935,134 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
         is returned, unless ``return_type`` is None, in which case a NumPy
         array of axes is returned with the same shape as ``layout``.
         See the prose documentation for more.
-
-    `**kwds` : Keyword Arguments
+    kwds : Keyword Arguments (optional)
         All other plotting keyword arguments to be passed to
-        matplotlib's boxplot function
+        matplotlib's function.
 
     Returns
     -------
     lines : dict
     ax : matplotlib Axes
-    (ax, lines): namedtuple
+        (ax, lines): namedtuple
+
+    See Also
+    --------
+    matplotlib.pyplot.boxplot: Make a box and whisker plot.
 
     Notes
     -----
     Use ``return_type='dict'`` when you want to tweak the appearance
     of the lines after plotting. In this case a dict containing the Lines
     making up the boxes, caps, fliers, medians, and whiskers is returned.
-    """
 
+    Examples
+    --------
+    .. plot::
+        :context: close-figs
+
+        >>> np.random.seed(1234)
+
+        >>> df = pd.DataFrame({
+        ...     u'stratifying_var': np.random.uniform(0, 100, 20),
+        ...     u'price': np.random.normal(100, 5, 20),
+        ...     u'demand': np.random.normal(100, 10, 20)})
+
+        >>> df[u'quartiles'] = pd.qcut(
+        ...     df[u'stratifying_var'], 4,
+        ...     labels=[u'0-25%%', u'25-50%%', u'50-75%%', u'75-100%%'])
+
+        >>> df
+            stratifying_var       price      demand quartiles
+        0         19.151945  106.605791  108.416747     0-25%%
+        1         62.210877   92.265472  123.909605    50-75%%
+        2         43.772774   98.986768  100.761996    25-50%%
+        3         78.535858   96.720153   94.335541   75-100%%
+        4         77.997581  100.967107  100.361419    50-75%%
+        5         27.259261  102.767195   79.250224     0-25%%
+        6         27.646426  106.590758  102.477922     0-25%%
+        7         80.187218   97.653474   91.028432   75-100%%
+        8         95.813935  103.377770   98.632052   75-100%%
+        9         87.593263   90.914864  100.182892   75-100%%
+        10        35.781727   99.084457  107.554140     0-25%%
+        11        50.099513  105.294846  102.152686    25-50%%
+        12        68.346294   98.010799  108.410088    50-75%%
+        13        71.270203  101.687188   85.541899    50-75%%
+        14        37.025075  105.237893   85.980267    25-50%%
+        15        56.119619  105.229691   98.990818    25-50%%
+        16        50.308317  104.318586   94.517576    25-50%%
+        17         1.376845   99.389542   98.553805     0-25%%
+        18        77.282662  100.623565  103.540203    50-75%%
+        19        88.264119   98.386026   99.644870   75-100%%
+
+    To plot the boxplot of the ``demand`` just put:
+
+    .. plot::
+        :context: close-figs
+
+        >>> boxplot = df.boxplot(column=u'demand', by=u'quartiles')
+
+    Use ``grid=False`` to hide the grid:
```

Comment:
Just changing this argument doesn't look like a lot of changes to justify an example. Perhaps we can change multiple parameters at the same time to show a more interesting example.